apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -160,14 +160,14 @@ type Minio struct {
 	// +optional
 	// +kubebuilder:validation:Pattern=^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
 	StorageSize string `json:"storageSize,omitempty"`
-	// Defines the number of Minio replicas.
-	// Supply 1 to run Minio in standalone mode with no redundancy.
-	// Supply 4 or more to run Minio in distributed mode.
-	// Note that it is not possible to upgrade Minio from standalone to distributed mode.
-	// Setting this will override the number of replicas set by 'Size'.
-	// More info: https://docs.min.io/docs/distributed-minio-quickstart-guide.html
+	// Defines the number of servers (pods) to launch in minio. Keep in mind that the number of
+	// volumes*servers should be >=4.
 	// +optional
-	Replicas int32 `json:"replicas,omitempty"`
+	Servers int32 `json:"servers,omitempty"`
+	// Defines how many volumes each server should have. Keep in mind that the number of volumes*servers
+	// should be >= 4.
```

Comment:
Is mandatory. [ref](https://github.com/minio/operator/blob/master/pkg/apis/minio.min.io/v2/types.go#L577)

Reply:
Do we have any validation? What will happen If I set

```
replicas: 1
servers: 1
```

Reply:
We do not. Most likely the operator will fail in some way, though they say they only let `>= servers * volumesPerServer > 4` but I have used it with 1 server and 4 volumes locally. Since this is not recommended for production and only for development purposes I didn't put much thought into it.